docs/scenarios/library-authoring/pinning-libraries-to-rc2.md

```diff
@@ -0,0 +1,70 @@
+# How to manage package dependency versions for RC2
+
+By [Phillip Carter](https://github.com/cartermp)
+
+This article covers how to manage package versions when authoring a library for .NET Core.
+
+## Glossary
+
+**TFM** - Target Framework Moniker.  This is a string that represents the target you want your code to run on.  In this case of a library, this is `netstandard1.x`, where `1 <= x <= 5`.
+
+**Metapackage** - A NuGet package that represents a set of NuGet packages.  They're organized in a way that makes sense for general library and app development.
+
+## Pin your code to RC2
```

Comment:
I'm not sure I would us the term "pin" through out this document as that has negative connotation for some and they feel like the are trapped or stuck with a particular version. Perhaps a better work would be "snap" in this document.